src/Bicep.Decompiler.IntegrationTests/Working/copyloop/main.bicep

```diff
@@ -0,0 +1,57 @@
+param storageCount int = 2
+param dataDiskCount int = 2
+param vmCount int = 2
+param storagePrefix string
+param vmPrefix string
+param vmSize string
+
+@secure()
+param adminPassword string
+
+resource storagePrefix_id 'Microsoft.Storage/storageAccounts@2019-04-01' = [for i in range(0, storageCount): {
+  name: toLower(concat(i, storagePrefix, uniqueString(resourceGroup().id)))
+  location: resourceGroup().location
+  sku: {
+    name: 'Standard_LRS'
+  }
+  kind: 'Storage'
+  properties: {}
+}]
+
+resource vmPrefix_resource 'Microsoft.Compute/virtualMachines@2020-06-01' = [for i in range(0, vmCount): {
```

Comment:
>range(0, vmCount) [](start = 86, length = 17)

Unrelated to the PR, but the frequency of occurence of this pattern makes me wonder if we should have syntactic sugar for the range function. Maybe `0..vmCount` would compile to `range(0, vmCount)`?

Reply:
Note that I'm always codegenning `range(...)` with this PR regardless of whether iterating over a collection of things, or truly trying to do things `x` times (since there's no way to express the former in ARM).

I was thinking about adding a syntax rewriter to detect and replace any scenarios where we could replace with a collection iterator, but didn't want to include that in this PR. When I converted the loop templates in the quickstart repo, I got the impression that iterating over a collection was a much more common pattern.